migration-guides/1.x-2.0.adoc

```diff
@@ -0,0 +1,205 @@
+
+| OSGi Plugin
+| no longer maintained (https://github.com/jqassistant-archive/jqa-osgi-plugin[archive])
+
+| PlantUML Report Plugin
+| https://github.com/jqassistant-plugin/jqassistant-plantuml-report-plugin[]
+
+| RDBMS Plugin
+| https://github.com/jqassistant-plugin/jqassistant-rdbms-plugin[]
+
+| Spring Plugin
+| https://github.com/jqassistant-plugin/jqassistant-spring-plugin[]
+
+|===
+
+
+Refer to the documentation of each plugin for the coordinates.
+
+NOTE: Declaring jQAssistant plugins as dependency of the jQAssistant Maven plugin in a `pom.xml` __might__ still work but is no longer supported. Such declarations must be migrated to the `plugins` section of the `.jqassistant.yml` file.
+
+=== Configure Plugin Repositories (Command Line Utility)
+
+Plugins are now longer distributed with the command line distribution. Declared plugins (see chapter before) including their required dependencies are resolved and downloaded on startup from Maven repositories (defaulting to Maven Central).
+
+For specific setups (e.g. in corporate environments) an existing Maven configuration in the user's home directory (`~/.m2/settings.xml`) is used automatically.
+
+Alternatively the proxy and repositories can be specified in the file `.jqassistant.yml`:
+
+[source,yaml]
+.jqassistant.yml
+----
+jqassistant:
+  repositories:
+    local: ~/.jqassistant/repository # local repository, caching downloaded plugins
+    proxy:
+      protocol: https
+      host: proxy.acme.com
+      port: 80
+      username: user
+      password: secrect
+    remotes:
+      corporate-repo:
+        url: https://corporate-repo.acme.com
+        username: user
+        password: secrect
+
+  plugins:
+    - group-id: org.jqassistant.plugin
+      artifact-id: jqassistant-spring-plugin
+      version: 2.0.0
+    - group-id: org.jqassistant.plugin
+      artifact-id: jqassistant-plantuml-report-plugin
+      version: 2.0.1
+----
+
+=== Activate Or Migrate Asciidoc Rules
+
+Support for embedding rules in Asciidoc files (.adoc) has been removed from the core distribution. It can be activated by declaring the Asciidoc Report Plugin in the configuration:
+
+[source,yaml]
+.jqassistant.yml
+----
+jqassistant:
+  plugins:
+    - group-id: org.jqassistant.plugin
+      artifact-id: jqassistant-asciidoc-report-plugin
+      version: 2.0.0
+    - group-id: org.jqassistant.plugin
+      artifact-id: jqassistant-plantuml-report-plugin # Often used together with the Asciidoc plugin
+      version: 2.0.1
+----
+
+NOTE: It is highly recommended to migrate rules embedded Asciidoc files to XML. The integration with Asciidoc is handled by the https://github.com/jqassistant-tooling/jqassistant-asciidoctorj-extensions[jQAssistant Asciidoc Extensions] which allow better interoperability with existing tool chains (e.g. http://doctoolchain.org/docToolchain/v2.0.x/[docToolchain] and https://confluence-publisher.atlassian.net/wiki/spaces/CPD/overview[Confluence Publisher])
+
```

Comment:
Hm, I think the description provided here is to short for the typical user. When they used AsciiDoc for documentation before, they want to use it also now. So a complete setup should b provided to keep frustration low. That should be extended to something like the following aspects:
1. Extract Groups, Concepts, and Constraints from AsciiDoc to XML (link 101 where this setup is shown)
2. Setup asciidoctorj for AsciiDoc rendering
3. Explain how jQA macros work in that context